genesyscloud/user_group_role_common.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 var (
-	roleAssignmentResource = &schema.Resource{
+	RoleAssignmentResource = &schema.Resource{
```

Comment:
move this file as well to its own package. most likely this wil be used by users and groups. and add a proxy for this.